cmd/root_test.go

```diff
@@ -55,6 +56,7 @@ func Test_runRoot(t *testing.T) {
 	}
 	defer os.RemoveAll(dir)
 	svr := startHTTPServer()
+	time.Sleep(1)
```

Comment:
why?

Reply:
The server take a bit more to start and the test fail because they don't find the server running. maybe there is a prettier way to do it?

Reply:
Maybe write a function that keeps making a `HEAD` request against this server and exits when either the server is ready or too much time passed

Reply:
Will do